metrics/web/lib/dashboard/presentation/widgets/metrics_table.dart

```diff
@@ -37,9 +37,17 @@ class MetricsTable extends StatelessWidget {
                   if (projects == null) return const LoadingPlaceholder();
 
                   if (projects.isEmpty) {
-                    return const _DashboardTablePlaceholder(
-                      text: DashboardStrings.noConfiguredProjects,
-                    );
+                    final projectNameFilter =
+                        projectsMetricsNotifier.projectNameFilter;
+
+                    if (projectNameFilter != null &&
+                        projectNameFilter.isNotEmpty) {
+                      return Container();
+                    } else {
+                      return const _DashboardTablePlaceholder(
+                        text: DashboardStrings.noConfiguredProjects,
+                      );
+                    }
```

Comment:
Maybe it will be more readable? What do you think? 
```suggestion
                                       final hasProjectNameFilter = projectNameFilter != null &&
                        projectNameFilter.isNotEmpty;

                    if (hasProjectNameFilter) return Container();

                    return const _DashboardTablePlaceholder(
                      text: DashboardStrings.noConfiguredProjects,
                    );
```